scripts/cli.ts

```
@@ -215,72 +287,24 @@ function copyFilesFromSourceSkill(sourceSkillPath: string, outputPath: string) {
-
-	// Manual skills
-	for (const name of manual) {
-		expected.add(name);
-	}
-
-	return expected;
-}
-
-function getGitSha(repositoryPath: string): null | string {
-	return execSafe("git rev-parse HEAD", repositoryPath);
+	return content.includes(`path = ${path}`);
```

ESLint

[error] Unsafe return of a value of type error.

(ts/no-unsafe-return)